### PR TITLE
feat: add --include-direct

### PR DESCRIPTION
**Pull request by joneugster**

### CODE REVIEW

**Comment by joneugster** [comment]
With the current logic there might be arrows between these first-degree imports if they directly import each other, but there are no arrows if one indirectly imports another...

File: ImportGraph/Cli.lean [hunk 1]
```diff
@@ -77,9 +83,18 @@ def importGraphCLI (args : Cli.Parsed) : IO UInt32 := do
     let toModule := ImportGraph.getModule to
     let includeLean := args.hasFlag "include-lean"
     let includeStd := args.hasFlag "include-std" || includeLean
-    let includeDeps := args.hasFlag "include-deps" || args.hasFlag "mark-module" || includeStd
+    let includeDeps := args.hasFlag "include-deps" || includeStd
+
+    -- `directDeps` are excempt from being filtered out.
```
Comment (semorrison) [inline ImportGraph/Cli.lean:88]:
```suggestion
    -- `directDeps` are exempt from being filtered out.
```

File: Main.lean [hunk 2]
```diff
@@ -21,7 +21,7 @@ def graph : Cmd := `[Cli|
     "from" : ModuleName;  "Only show the downstream dependencies of the specified module."
     "exclude-meta";       "Exclude any files starting with `Mathlib.[Tactic|Lean|Util|Mathport]`."
     "include-deps";       "Include used files from other libraries (not including Lean itself and `std`)"
-    "mark-module";        "Visually highlight the current module. (implies `--include-deps`)"
+    "mark-module";        "Include directly imported files from other libraries and visually highlight the current module"
```
Comment (semorrison) [inline Main.lean:24]: What if we split this? The name is a bit confusing by now.

How about "include-direct" (or "include-direct-deps", I don't mind), and then separately (perhaps below all the "include" options in the list) "mark-module", which only affects display. (Although I don't mind it if implied either "include-deps" or "include-direct".)

Comment (joneugster) [reply Main.lean:24]: ok I'll do that! I did think about the same, but I was initially hesitant adding a gazillion options

Comment (joneugster) [reply Main.lean]: done.

**Comment by joneugster** [comment]
Here is the current behaviour now.

Q: The implication `--mark-module ==> --include-direct-deps` means there are some nodes like `Lean` in the second image. Is this fine?

Image 1: `--mark-module`
mark-module

Image 2: `--mark-module --include-deps`
include-deps

**Comment by joneugster** [comment]
Image 1: `--mark-module`
mark-module

Image 2: `--mark-module --include-direct-deps`
include-direct-deps

Image 3: `--mark-module --include-deps`
include-deps


Image 3: `--mark-module --include-deps --include-direct-deps`
include-both

**Comment by joneugster** [comment]
Image 1: `--mark-module`
mark-module

Image 2: `--mark-module --include-direct-deps`
include-direct-deps

Image 3: `--mark-module --include-deps`
include-deps

Image 4: `--mark-module --include-direct-deps --include-deps`
include-both

**Pull request merged**